### PR TITLE
Add servers to global list

### PR DESCRIPTION
**Pull request by Darkness6030**
Mindurka.ru - HUB
Mindurka.ru:3000 - ATTACK
Mindurka.ru:4000 - PVP

### CODE REVIEW

**Comment by VadrixMashina** [comment]
Yes

**Comment by blackosp54** [comment]
Normal server, deserves a global!

**Review by misaka0954 — APPROVED** [review]

**Comment by RebornTrack970** [comment]
Why, what is the thing with everyone trying to add Vanilla servers?
Cant you just be or original?
Isnt there already like 7 vanilla servers?
Thats just deviding the player base...

**Comment by unjown** [comment]
is this another ru server?

**Comment by MindustryRu** [comment]
@unjown 
No, these are not our servers.

**Comment by KotMilk** [comment]
Good server

**Comment by KotMilk** [comment]
> @unjown
> Нет, это не наши серверы.

Ygy

**Comment by Darkness6030** [comment]
We have also added Survival and Sandbox servers to our HUB!
You should see 'em!

**Review by RebornTrack970 — CHANGES_REQUESTED** [review]
You should only add the hub as you can go to others from there.

**Comment by blackosp54** [comment]
> это еще один сервер mindustry.ru?

your logic goes off scale, don't you really understand what.RU it's just a domain

**Comment by Darkness6030** [comment]
> You should only add the hub as you can go to others from there.

Hmm, i forgot

File: servers_v6.json [hunk 1]
```diff
@@ -86,5 +86,9 @@
   {
     "name": "Nydus",
     "address": ["v6.mindustry.nydus.app:6566"]
+  },
+  {
+    "name": "Darkdustry",
+    "address": ["mindurka.ru", "mindurka.ru:3000", "mindurka.ru:4000"]
```
Comment (RebornTrack970) [inline servers_v6.json:92]: Please only add the hub.

Comment (Darkness6030) [reply servers_v6.json]: Done

**Comment by RebornTrack970** [comment]
Much better.

**Review by RebornTrack970 — APPROVED** [review]

**Pull request merged**